hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
Ah ok, that makes sense - if the zeta method can work on a line scan should it not be possible for cross-section method to work on a line scan too? I don't know if that's a stupid question, sorry if it is :)